CHANGES.md

```diff
@@ -60,6 +60,13 @@ To be released.
 
 ### Backward-incompatible storage format changes
 
+ -  Now Block and Transaction will be stored in distributed database.
+    Each block and transaction is divided and saved in epoch units
+    according to the unix timestamp that you have.
+    This epoch is divided using the seconds set in
+    `txAtEachEpoch` parameter and `blockAtEachEpoch` parameter
+    from `RocksDBStore`. (The default is 86400.) [[#1183], [#1194]]
```

Comment:
The term “distribution” might have a nuance that these are distributed over network.  I suggest you to choose any other terms like “partitioning” or “sharding.”

```suggestion
 -  (Libplanet.RocksDBStore) The blocks and transactions became stored in
    multiple databases.  Each block and transaction belongs to a partition
    of the database, according to its epoch unit, which is its Unix timestamp.
    Every epoch is divided by certain seconds, configured by `RocksDBStore()`
    constructor's `txAtEachEpoch` and `blockAtEachEpoch` parameters (86400
    by default).  [[#1183], [#1194]]
```

Reply:
Changed PR title and description too.